### ISSUE TITLE
Should TTA strategy be used when generating detection results on the training dataset for training the refine model?

### ISSUE THREAD

**Issue by qinhuan** [PJLab-ADG/DetZero#32]

**Comment by SISTMrL** [comment]
你好，可以加个联系方式吗我最近也在搞这个优化模块，可以交流下谢谢，我的微信：15256956711

**Comment by qinhuan** [comment]
I reproduced the accuracy in the paper without using tta
